### PR TITLE
code migration

### PR DESCRIPTION
**Pull request by BC-Lizard**
Another webhook test. Feel free to leave a review but no need to actually complete the merge.

### CODE REVIEW

**Comment by BC-Lizard** [comment]
This is a test comment

File: Angular/karma.conf.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    },
+    jasmineHtmlReporter: {
+      suppressAll: true // removes the duplicated traces
+    },
+    coverageReporter: {
+      dir: require('path').join(__dirname, './coverage/Angular'),
+      subdir: '.',
+      reporters: [
+        { type: 'html' },
+        { type: 'text-summary' }
```
Comment (markmoorerev) [inline Angular/karma.conf.js:32]: don't forget to add "lcov" as a type.

Comment (BC-Lizard) [reply Angular/karma.conf.js:32]: Thanks for the review. This is a test reply ✔️

File: Logic/ReviewLogic.cs [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            List<Review> reviews = new List<Review>();
+
+            for (int i = startIndex; i <= endIndex; i++)
+            {
+                reviews.Add(Mapper.RepoReviewToReview(repoReviews[i]));
+            }
+            return reviews;
+        }
+
+        public async Task<bool> SetReviewsPageSize(int pagesize)
```
Comment (markmoorerev) [inline Logic/ReviewLogic.cs:115]: needs xml comments

**Review by markmoorerev — APPROVED** [review]

**Pull request closed without merging**